xml/System.Net.Sockets/Socket.xml

```diff
@@ -10325,7 +10315,11 @@ This member outputs trace information when you enable network tracing in your ap
         </remarks>
         <exception cref="T:System.IO.FileNotFoundException">The file specified in the <see cref="P:System.Net.Sockets.SendPacketsElement.FilePath" /> property was not found.</exception>
         <exception cref="T:System.InvalidOperationException">A socket operation was already in progress using the <see cref="T:System.Net.Sockets.SocketAsyncEventArgs" /> object specified in the <paramref name="e" /> parameter.</exception>
-        <exception cref="T:System.NotSupportedException">Windows XP or later is required for this method. This exception also occurs if the <see cref="T:System.Net.Sockets.Socket" /> is not connected to a remote host.</exception>
+        <exception cref="T:System.NotSupportedException">The operating system is not Windows XP or later.
```

Comment:
@antonfirsov Is the OS requirement still applicable here?

Reply:
If I get #5453 / #2053 right, this can be removed, since there are no documented OS-es this applies to.